libraries/rush-sdk/src/index.ts

```diff
@@ -61,8 +63,48 @@ if (rushLibModule === undefined) {
 
 // SCENARIO 3:  A tool or script depends on "rush-sdk", and is meant to be used inside a monorepo folder.
 // In this case, we can use install-run-rush.js to obtain the appropriate rush-lib version for the monorepo.
-//
-// NOT IMPLEMENTED YET
+if (rushLibModule === undefined) {
+  try {
+    const rushJsonPath: string | undefined = tryFindRushJsonLocation(process.cwd());
+    if (!rushJsonPath) {
+      throw new Error('Could not find rush.json');
+    }
+    const monorepoRoot: string = path.dirname(rushJsonPath);
+
+    const rushJson: JsonObject = JsonFile.load(rushJsonPath);
+    const { rushVersion } = rushJson;
+
+    const installRunNodeModuleFolder: string = path.join(
+      monorepoRoot,
+      `common/temp/install-run/@microsoft+rush@${rushVersion}`
+    );
+
+    try {
+      // First, try to load the version of "rush-lib" that was installed by install-run-rush.js
+      rushLibModule = requireRushLibUnderFolderPath(installRunNodeModuleFolder);
+    } catch (e) {
+      try {
+        const installAndRunRushJSPath: string = path.join(monorepoRoot, 'common/scripts/install-run-rush.js');
+        Executable.spawnSync('node', [installAndRunRushJSPath, '--help'], {
+          stdio: 'ignore'
```

Comment:
This output is important, however I agree that it may be rather noisy on the console.

Maybe what we could do is save it to a text string, and then print it to `console.error()` only if the operation fails.